Earthfile

```diff
@@ -215,12 +215,8 @@ framework:
     # Copy flavor-specific overlay files
     IF [[ "$FLAVOR" =~ ^alpine* ]]
         COPY overlay/files-alpine/ /framework
-    ELSE IF [ "$FLAVOR" = "fedora" ] || [ "$FLAVOR" = "rockylinux" ]
-        COPY overlay/files-fedora/ /framework
```

Comment:
we don't need the fedora files anymore?

Reply:
probably yes, trying to find out if selinux works now on fedora+ubuntu....seems to work on some ubuntus somehow?